pkg/controller/direct/compute/forwardingrule_controller.go

```diff
@@ -333,11 +334,14 @@ func (a *forwardingRuleAdapter) Create(ctx context.Context, createOp *directbase
 	if err != nil {
 		return fmt.Errorf("creating ComputeForwardingRule %s: %w", a.fullyQualifiedName(), err)
 	}
-	err = op.Wait(ctx)
-	if err != nil {
-		return fmt.Errorf("waiting ComputeForwardingRule %s create failed: %w", a.fullyQualifiedName(), err)
+	if !op.Done() {
+		err = op.Wait(ctx)
+		if err != nil {
+			return fmt.Errorf("waiting ComputeForwardingRule %s create failed: %w", a.fullyQualifiedName(), err)
+		}
```

Comment:
Mind explaining what the GCP level difference is for this change?

Reply:
I think I know this one from chatting with @gemmahou ; the issue is that we don't want to get the operation if it is already done - we don't want an extra operation get in the http log.

Reply:
Right!